schema/gosling.schema.json

```diff
@@ -448,14 +448,9 @@
         },
         "scaleOffset": {
           "description": "Whether to use offset of the domain proportionally. This is bound to brushes on the color legend. __Default__: `[0, 1]`",
-          "items": [
-            {
-              "type": "number"
-            },
-            {
-              "type": "number"
-            }
-          ],
+          "items": {
```

Comment:
Changes in `schema.json` files seem to be due to the upgrade of `ts-json-schema-generator`